db/seeds.rb

```diff
@@ -21,4 +25,9 @@
 
 Video.create(title: "Southpark", description: "Cartman and friends.", large_cover_url: "south_park.jpg", small_cover_url: "south_park_small.jpg", category_id: 2)
 Video.create(title: "Ray Donovan", description: "fixes problems of the rich and famous in Hollywood", small_cover_url: "ray_donovan_small.jpg", large_cover_url: "ray_donovan.jpg", category_id: 1)
-Video.create(title: "Better Call Saul", description: "The life of the lawyer from Breaking Bad, before meeting Walter White", small_cover_url: "better_call_saul_small.jpg", large_cover_url: "better_call_saul.jpg", category_id: 1)
\ No newline at end of file
+Video.create(title: "Better Call Saul", description: "The life of the lawyer from Breaking Bad, before meeting Walter White", small_cover_url: "better_call_saul_small.jpg", large_cover_url: "better_call_saul.jpg", category_id: 1)
```

Comment:
I noticed you hardcode id's here like `category_id: 1` I would recommend just referring to existing category that will be stored in a variable.